flow/events/get/get.go

```diff
@@ -0,0 +1,98 @@
+/*
+ * Flow CLI
+ *
+ * Copyright 2019-2020 Dapper Labs, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package get
+
+import (
+	"log"
+	"strconv"
+	"strings"
+
+	"github.com/psiemens/sconfig"
+	"github.com/spf13/cobra"
+
+	cli "github.com/onflow/flow-cli/flow"
+)
+
+type Config struct {
+	Host    string `flag:"host" info:"Flow Access API host address"`
+	Verbose bool   `flag:"verbose" info:"Verbose output"`
+}
+
+var conf Config
+
+var Cmd = &cobra.Command{
+	Use:   "get <event_name> <block_height_range_start> <optional:block_height_range_end|latest>",
```

Comment:
Maybe use "type ID" instead of "name", as someone might assume it's possible to just specify e.g `AccountCreated`. 

Also, maybe make it clearer that `latest` here is a literal, not a variable like `block_height_range_start` and `block_height_range_end`

```suggestion
	Use:   "get <event_type_id> <block_height_range_start> <optional:block_height_range_end|'latest'>",
```